app-drawer/app-drawer.html

```diff
@@ -196,26 +196,26 @@
       },
 
       listeners: {
-        track: '_trackHandler'
+        track: '_track',
+        transitionEnd: '_transitionEnd',
+        webkitTransitionEnd: '_transitionEnd'
```

Comment:
I think you need to lowercase the event name `transitionendend` and it should work on Chrome, Safari and other supported browsers.
```
transitionend: '_transitionEnd'
```